hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveMetastore.java

```diff
@@ -112,6 +113,9 @@ public void start(HiveConf conf) {
   public void start(HiveConf conf, int poolSize) {
     try {
       this.hiveLocalDir = createTempDirectory("hive", asFileAttribute(fromString("rwxrwxrwx"))).toFile();
+      // Ideally we can delete the folder in stop(), but tests would fail if we do that, probably because we are not
```

Comment:
I would prefer to fix or at least find the underlying issue.
Otherwise the files which are carried over will cause issues which are really hard to debug

Reply:
OK, I'll look into that.